### PR TITLE
nodejs create new package documentation

### PR DESCRIPTION
**Pull request by dLobatog**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,6 +132,42 @@ You'll also need an alias `kojikat` to point to:
...
+[npm2rpm](https://www.npmjs.com/package/npm2rpm#packaging-npm-modules-with-bundled-dependencies--s---stategy-bundle)
+
+In both cases:
+
+1. Install npm2rpm: `npm install -g npm2rpm`
+1. Checkout a branch 'rpm/example-version' or similar (the rpm/ part is important)
+  * `mkdir nodejs-example`
+  * `cd nodejs-example`
+1. Generate the spec and download the sources automatically with npm2rpm.
+  * For packags without dependencies - `npm2rpm -n example -v version -s single`
```
Comment (domcleal) [inline README.md]: "packages", ditto on the line below

File: README.md [hunk 2]
```diff
@@ -127,6 +132,42 @@ You'll also need an alias `kojikat` to point to:
...
+  * For packags that bundle dependencies - `npm2rpm -n example -v version -s bundle`
+1. This should have created two directories npm2rpm/SPECS and npm2rpm/SOURCES.
+   Move the content of them both to your nodejs-example directory and remove the npm2rpm folder.
+  * mv npm2rpm/SPECS/\* .
+  * mv npm2rpm/SOURCES/\* .
+  * rm -r npm2rpm
+1. If building a package with bundled dependencies, add the binary with the npm cache directly to git.
+  * `git add example-version-registry.npmjs.org.tgz
```
Comment (domcleal) [inline README.md]: missing end `

File: README.md [hunk 3]
```diff
@@ -127,6 +132,42 @@ You'll also need an alias `kojikat` to point to:
...
+1. `git-annex` the rest of the sources
+  * `git annex add \*.tgz`
```
Comment (domcleal) [inline README.md]: remove the escape when within the `

**Comment by dLobatog** [comment]
Updated, thanks for taking a look. The output of npm2rpm I refer to here might not be the one that you get from the version that's now on npm, but it should work with the latest branch of https://github.com/dlobatog/npm2rpm . I plan on releasing a new version once the whole thing is merged so that I can be sure it works like that

**Comment by domcleal** [comment]
Thanks @dLobatog.

**Pull request merged**